apps/deploy-web/env/.env

```
@@ -34,6 +34,6 @@ NEXT_PUBLIC_UNLEASH_APP_NAME=console-web
 SENTRY_ORG=cloudmos
 SENTRY_PROJECT=deploy-web
 
-NEXT_PUBLIC_AMPLITUDE_PROXY_URL="/api/collect"
+NEXT_PUBLIC_AMPLITUDE_PROXY_URL="https://console-proxy.akash.network/collect"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)